qds_sdk/clusterv2.py

```diff
@@ -33,9 +33,22 @@ def parsers(action):
             ClusterCmdLine.create_update_clone_parser(clone, action="clone")
             clone.set_defaults(func=ClusterV2.clone)
 
+        if action == "list":
+            li = subparsers.add_parser("list", help="list clusters from existing clusters depenging upon state")
+            ClusterCmdLine.list_parser(li, action="list")
+            li.set_defaults(func=ClusterV2.list)
         return argparser
 
     @staticmethod
+    def list_parser(subparser, action=None):
+
+        # cluster info parser
+        ClusterInfoV2.list_info_parser(subparser, action)
+
+        # engine config parser
+        Engine.engine_parser(subparser)
```

Comment:
Why is engine parser required in listing cluster? As far as I understand, listing of clusters should only require passing cluster id , label or cluster state.